dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -2377,44 +2396,33 @@ BitmapFilterPtr Segment::buildBitmapFilterStableOnly(const DMContext & dm_contex
 {
     Stopwatch sw;
     const auto & dmfiles = segment_snap->stable->getDMFiles();
-    RUNTIME_CHECK(!dmfiles.empty(), dmfiles.size());
-    std::vector<std::vector<PackInfo>> all_packs{dmfiles.size()};
-    std::vector<IdSetPtr> some_packs{dmfiles.size()};
-    bool all_match = true;
-    size_t some_packs_count = 0;
-    for (size_t i = 0; i < dmfiles.size(); i++)
-    {
-        std::tie(all_packs[i], some_packs[i]) = parseDMFilePackInfo(dmfiles[i],
-                                                                    dm_context,
-                                                                    read_ranges,
-                                                                    filter,
-                                                                    max_version);
-        all_match = all_match && (all_packs[i].size() == dmfiles[i]->getPacks());
-        some_packs_count += some_packs[i]->size();
-    }
+    RUNTIME_CHECK(!dmfiles.empty());
+
+    auto [all_packs, some_packs] = parseDMFilePackInfo(dmfiles, dm_context, read_ranges, filter, max_version);
```

Comment:
may be we can find a better name for "all_packs and some_packs"?

Reply:
done